tensorboard/plugins/image/summary_v2.py

```diff
@@ -35,7 +70,8 @@ def image(name, data, step=None, max_outputs=3, description=None):
         width of the images, and `c` is the number of channels, which
         should be 1, 2, 3, or 4 (grayscale, grayscale with alpha, RGB, RGBA).
         Any of the dimensions may be statically unknown (i.e., `None`).
-        Floating point data will be clipped to the range [0,1).
+        Floating point data will be clipped to the range [0,1]. Other data types
+        may be clipped into an allowed range for safe casting to uint8.
```

Comment:
s/may be/will be/ - I think better to be definitive here.  We can even always give a pointer to `convert_image_dtype` if people want to know exactly what we do.

Reply:
Done, and mentioned `convert_image_dtype`.